infra/database/type/opengauss/src/main/java/org/apache/shardingsphere/infra/database/opengauss/metadata/data/loader/OpenGaussMetaDataLoader.java

```diff
@@ -70,7 +72,10 @@ public final class OpenGaussMetaDataLoader implements DialectMetaDataLoader {
     @Override
     public Collection<SchemaMetaData> load(final MetaDataLoaderMaterial material) throws SQLException {
         try (Connection connection = material.getDataSource().getConnection()) {
-            Collection<String> schemaNames = SchemaMetaDataLoader.loadSchemaNames(connection, TypedSPILoader.getService(DatabaseType.class, "openGauss"));
+            DatabaseType databaseType = TypedSPILoader.getService(DatabaseType.class, "openGauss");
+            DialectDatabaseMetaData dialectDatabaseMetaData = new DatabaseTypeRegistry(databaseType).getDialectDatabaseMetaData();
+            dialectDatabaseMetaData.getSchema(connection);
```

Comment:
This method is called, but the return value is not processed. What is the purpose of this?

Reply:
the method getSchema() is overrided in Class OpenGaussDatabaseMetaData, which will set defaultSchema first.
the code like this:
```
    @Override
    public String getSchema(final Connection connection) {
        this.defaultSchema = DialectDatabaseMetaData.super.getSchema(connection);
        return this.defaultSchema;
    }
```

Reply:
In the DialectDatabaseMetaData interface, the most appropriate method to override for retrieving the schema based on connection parameters is getSchema